js/pages/program_page/components/indicator_list.js

```diff
@@ -286,17 +287,27 @@ export class IndicatorListTable extends React.Component {
 
 
 const IndicatorListTableButtons = observer(function ({program, rootStore, ...props}) {
+    let chosenTiers = [];
+    rootStore.levelTiers.map((level, i) => chosenTiers[i] = level.name ); 
```

Comment:
Can you consolidate these lines so it's like this?
`let chosenTiers = rootStore.levelTiers.map((level, i) => chosenTiers[i] = level.name ); `

Reply:
Cool, I made that change.